generateBins.py

```diff
@@ -120,15 +177,25 @@ def save(pfpr, treatments, filename, username):
         gisPath = str(sys.argv[3])
 
     # Process and print the relevent ranges for the user
-    [ pfpr, treatments ] = process(configuration, gisPath)
+    [ pfpr, treatments ] = process( configuration, gisPath)
     for zone in pfpr.keys():
         print("Climate Zone {}".format(zone))
         print("Treatments: {}".format(sorted(treatments[zone])))
         print("Populations: {}".format(sorted(pfpr[zone].keys())))
         for popBin in sorted(pfpr[zone].keys()):
             print("{} - {} to {} PfPR".format(popBin, min(pfpr[zone][popBin]), max(pfpr[zone][popBin])))
-        print
+        #print
 
     # Save the basic script
     if not os.path.isdir('out'): os.mkdir('out')
     save(pfpr, treatments, 'out/calibration.sh', username)
+
+    # GVF Implementation
+    gvf = 0.0
+    nclasses = 20
+    while gvf < 0.8:
+        gvf = goodness_of_variance_fit(array_1d, nclasses)
+        nclasses += 1
+    print("The value of Goodness of Variance fit is:", gvf)
+    if (gvf < 0.7):
+        print("Warning: GVF too low")
```

Comment:
I see what you are doing here, so this is a good start, but this needs to be shifted up into `process` and then then the appropriate bins used instead of the hardcoded `POPULATION_BINS`.